### ISSUE TITLE
Layer methods / missing permissions from server response

### ISSUE THREAD

**Issue by francoisromain** [camomile-project/camomile-server#80]
The response of method `getLayers` with a `id_corpus` parameter misses permissions: 
- in `permissions.users`: there is only the current user (even if has `ADMIN` permission on that layer and should see permissions for all users)
- in `permissions.groups`: there are only the groups which the current user belongs (even if has `ADMIN` permission and should be able to see all groups permissions)

**Comment by hbredin** [comment]
Let me rephrase what you think should happen:
* if current user does not have `ADMIN` permission on the layer, `getLayers` should stick to the current behavior,
* if current has `ADMIN` permission , it should also see permissions for all users and groups.

I think it is indeed a good idea. 
Can you confirm my understanding is correct?

**Comment by francoisromain** [comment]
Yes exactly. Thank you
